op-scim-bridge/values.yaml

```diff
@@ -46,16 +49,12 @@ scim:
   # ingress defines the configuration for an ingress resource
   ingress:
     enabled: false
-    className: ""
-    hosts: []
-      # - host: chart-example.local
-      #   paths:
-      #     - path: /
-      #       pathType: ImplementationSpecific
-    tls: []
-      #  - secretName: chart-example-tls
-      #    hosts:
-      #      - chart-example.local
+    className:
+    domain:
+    path: \
+    tls:
```

Comment:
Note that this makes https://github.com/1Password/op-scim-helm/pull/60/files#diff-0de2f4f1369c8e7f265bf4aa8b3996820bfe58482ec75ee491212a670e29cbedR80-R82 redundant but I wasn't sure how you wanted to handle that as it may break things on your end.

Reply:
I'm afraid this link does not resolve anywhere obvious. I assume this is because it is outside the diff. Am I correct in thinking you are referring to `values.yaml` lines 80-82?

AKA

```go
tls:
    enabled: false
    secret: "{{ .Chart.Name }}-bridge-tls"
 ```